src/Resources/config/twig.xml

```diff
@@ -29,5 +29,10 @@
                 <argument>%sonata.admin.twig.extension.x_editable_type_mapping%</argument>
             </call>
         </service>
+	    <service id="sonata.templates.twig.extension" class="Sonata\AdminBundle\Twig\Extension\SonataTemplatesExtension" public="true">
```

Comment:
Does it really need to be public?

Reply:
I guess not. I copied it from the service above it. I'll change it.